tests/functional/views/basic_lti_launch_test.py

```diff
@@ -41,41 +40,10 @@ def test_db_configured_basic_lti_launch(
         assert js_config["mode"] == JSConfig.Mode.BASIC_LTI_LAUNCH
         assert urlencode({"url": assignment.document_url}) in js_config["viaUrl"]
 
-    @pytest.mark.parametrize(
-        "launch_params_fixture_name,existing_assignment_fixture_name",
-        [
-            param(
-                "url_launch_params",
-                None,
-                id="url launch, with no DB rows",
-            ),
-            param(
-                "url_launch_params",
-                "legacy_speedgrader_assignment",
-                id="url launch, with existing legacy SpeedGrader DB row",
-            ),
-            param(
-                "canvas_url_launch_params",
-                None,
-                id="canvas url launch, with no DB rows",
-            ),
-            param(
-                "canvas_url_launch_params",
-                "legacy_speedgrader_assignment",
-                id="canvas url launch, with existing legacy SpeedGrader DB row",
```

Comment:
All of these try to prove we are no falling to a trap with a bug in Canvas with the intention to read our DB to get the document_url on launch.

We never do that so  this test only add confusion.